sycl/test-e2e/AddressSanitizer/out-of-bounds/USM/parallel_for_double.cpp

```diff
@@ -0,0 +1,40 @@
+// REQUIRES: linux, cpu
```

Comment:
Should probably also require `aspect-fp64` here.

Reply:
Done